packages/mds-db/tests/mds-db.spec.ts

```diff
@@ -146,6 +147,38 @@ async function seedDB() {
   await MDSDBPostgres.seed({ devices, events, telemetry })
 }
 
+async function seedTripEvents() {
+  await MDSDBPostgres.reinitialize()
+  const devices: Device[] = makeDevices(9, startTime, JUMP_PROVIDER_ID) as Device[]
+  devices.push(JUMP_TEST_DEVICE_1 as Device)
+  const trip_id = uuid()
+  const deregisterEvents: VehicleEvent[] = makeEventsWithTelemetry(
```

Comment:
Why is this called `deregisterEvents` when these appear to be `trip_start` events?

Reply:
yeah, totally copypasta mistake. Thanks!